src/core/services/gesture/gesture.js

```diff
@@ -318,6 +331,19 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
       }
     });
 
+  function getTouchAction() {
+    var testEl = angular.element('<div>')[0];
+    var vendorPrefixes = ['', 'webkit', 'Moz', 'MS', 'ms', 'o'];
```

Comment:
Thought the same, but HammerJS used that as well. And I'm sure it has a significance